backends/platform/sdl/sdl.cpp

```diff
@@ -207,6 +208,16 @@ void OSystem_SDL::initBackend() {
 	if (_eventSource == 0)
 		_eventSource = new SdlEventSource();
 
+	if (_eventManager == nullptr) {
+		DefaultEventManager *eventManager = new DefaultEventManager(_eventSource);
+#if SDL_VERSION_ATLEAST(2, 0, 0)
```

Comment:
Is this going to cause problems for (possibly hypothetical at this point) backends that use SDL2 events but don’t use OSystem_SDL? Maybe this should be an `#if/else` on the member initialiser in the DefaultEventManager constructor instead to avoid such an issue.

Reply:
I'm reluctant to introduce platform specific code in `DefaultEventManager`. And introducing `Sdl2EventManager` just for this feels like over-engineering.

Reply:
That seems like a fair balance.